packages/react/src/components/AccountSettings/utils/displayText.ts

```diff
@@ -0,0 +1,18 @@
+export const defaultChangePasswordDisplayText = {
+  currentPasswordLabel: 'Current Password',
+  newPasswordLabel: 'New Password',
+  confirmPasswordLabel: 'Confirm Password',
+  updatePasswordText: 'Update password',
+};
+
+export type ChangePasswordDisplayText = typeof defaultChangePasswordDisplayText;
+
+export const defaultDeleteUserDisplayText = {
+  deleteAccountText: 'Delete Account',
+  warningText:
+    'Deleting your account is not reversible. You will lose access to your account and all data associated with it.',
+  cancelText: 'Cancel',
+  deleteMyAccountText: 'Delete my account',
```

Comment:
Think this would read better without the "My" -> `deleteAccountText`, and might make sense to remove the "my" from the value as well -> `'Delete Account'`

Reply:
We already have a `deleteAccountText` defined.

Reply:
Hmmm, that doesn't really impact the fact that "Delete my account" could read better?